packages/flutter/lib/src/foundation/node.dart

```diff
@@ -87,6 +87,9 @@ class AbstractNode {
   /// Subclasses with children should override this method to first call their
   /// inherited [attach] method, and then [attach] all their children to the
   /// same [owner].
+  ///
+  /// If you override this, make sure your method starts with a call to
+  /// `super.attach(owner)`.
```

Comment:
The Flutter API docs voice avoids telling the reader what to do -- because we don't know if the reader wrote the code they care about, or if they are just trying to learn how someone else's code works. So we use the passive voice or talk about the code itself, without reference to the reader or the programmer, as in "Implementations of this method should start with a call to the superclass, as in \`super.attach(owner)\`.".

Reply:
(comments along these lines apply below as well)

Reply:
(we should fix the existing text too)

Reply:
@Hixie Thanks, makes sense! Adjusted it in the new and existing docs.